pkg/version.go

```diff
@@ -16,6 +16,33 @@ limitations under the License.
 
 package pkg
 
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
 // VERSION is the version string for built artifacts. It's set by the build system, and should
 // not be changed in this codebase
 var VERSION = "UNKNOWN"
+
+// Version decides whether we should print the version and leave.
```

Comment:
Does this have an analog in k8s?  If so, we should try to get this moved into `k8s.io/apiserver` so we can reuse it without a dependency on k8s itself.